reporter/src/funTest/kotlin/reporters/StaticHtmlReporterTest.kt

```diff
@@ -35,6 +36,16 @@ class StaticHtmlReporterTest : WordSpec() {
     private val ortResult = File("src/funTest/assets/static-html-reporter-test-input.yml")
             .readValue<OrtResult>()
 
+    private fun generateReport(ortResult: OrtResult) =
+            ByteArrayOutputStream().also { outputStream ->
+                StaticHtmlReporter().generateReport(
+                        ortResult,
+                        DefaultResolutionProvider(),
+                        CopyrightGarbage(),
+                        outputStream
+                )
+            }.toString("UTF-8")
+
```

Comment:
I would prefer to have the init block above any function definitions, see:
https://kotlinlang.org/docs/reference/coding-conventions.html#class-layout

Except if this order is more consistent with our other test classes.